app/controllers/responsibilities_controller.rb

```diff
@@ -32,13 +32,13 @@ def create
 
     # GovUk notify can only deliver to 1 address at a time.
     emails.each do |email|
-      PomMailer.responsibility_override(
+      PomMailer.with(
         message: params[:message],
         prisoner_number: @responsibility.nomis_offender_id,
         prisoner_name: @offender.full_name,
         prison_name: @prison.name,
         email: email
-      ).deliver_later
+      ).new_allocation_email.deliver_later
```

Comment:
Should this be `responsibility_override` rather than `new_allocation_email`?

Reply:
!!!!!!